frida_tools/application.py

```diff
@@ -605,6 +606,9 @@ def work():
 
         return result[0]
 
+    def _on_device_found(self):
+        pass
```

Comment:
Would be good to move this to just before `_on_device_lost`.

Reply:
Fixed and rebased against main